amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
We have been doing an effort of adapting the test to be independent of the current status of the DB. I have the feeling this goes backwards with that. Also, if you run the tests twice in your machine and these tests fails, wouldn't that indicate that the problem is that there are tests in the repo that are not properly cleaning the environment?